biovectors_modules/word2vec_analysis_helper.py

```diff
@@ -75,53 +112,49 @@ def get_local_distance(
         shared_tokens - the list of all tokens shared across the years
         neighbors - the number of neighbor tokens to use
     """
+    token_distance = []
+    counter = 0
+    with Manager() as m:
+        tok_queue = m.JoinableQueue()
+        dist_queue = m.JoinableQueue()
```

Comment:
This looks very cool. I revisit this code the next time I need to run things in parallel.